src/components/Authentication/index.js

```diff
@@ -0,0 +1,4 @@
+import Logout from './Logout'
+import Authenticate from './Authenticate'
+
+export { Logout, Authenticate }
```

Comment:
what is the purpose of this file

Reply:
enable us to do `import { Logout }  from './Authentication';` instead of `import Logout from './Authentication/Logout';`

Or this, which makes more sense.
`import { Logout, Authenticate }  from './Authentication';` instead of
```
import Logout from './Authentication/Logout';
import Authenticate from './Authentication/Authenticate ';
```

Although I can't remember if this ever happens